core/bugnote_api.php

```diff
@@ -490,12 +490,6 @@ function bugnote_get_all_visible_bugnotes( $p_bug_id, $p_user_bugnote_order, $p_
 			# If the access level specified is not enough to see time tracking information
 			# then reset it to 0.
 			if( !$t_time_tracking_visible ) {
-				# If the time tracking is the only data in the note, then skip it.
-				if( is_blank( $t_bugnote->note ) ) {
```

Comment:
Shouldn't we just update this check to be empty note and no attached files instead of reverting this check?